packages/graphback-core/src/runtime/createInMemoryFilterPredicate.ts

```diff
@@ -114,6 +114,7 @@ const predicateMap: IPredicate = {
  * @param {QueryFilter} filter - subscription filter input object
  */
 export function createInMemoryFilterPredicate<T = any>(filter: QueryFilter): (input: Partial<T>) => boolean {
```

Comment:
```suggestion
export function createInMemoryFilterPredicate<T = any>(filter: QueryFilter = {}): (input: Partial<T>) => boolean {
```

Reply:
Yeap, this will work when filter is `undefined` but not cases where filter is `null`, probably won't happen but keeping in case users write their own filter and pass `null`.